libr/debug/p/native/w32.c

```diff
@@ -422,12 +415,6 @@ static int debug_exception_event (DEBUG_EVENT *de) {
 			get_w32_excep_name(code),
 			(int)de->dwThreadId);
 		break;
-#if __MINGW64__ || _WIN64
-	/* STATUS_WX86_BREAKPOINT */
```

Comment:
Why was this removed?

Reply:
Because case STATUS_WX86_BREAKPOINT will never happen here because w32_dbg_wait function already catch case STATUS_WX86_BREAKPOINT.